config.py

```diff
@@ -2,11 +2,11 @@
 
 
 # segmentation files suffix names
-axonmyelin_suffix = Path('_seg-axonmyelin.png') # axon + myelin segmentation suffix file name
-axon_suffix = Path('_seg-axon.png')             # axon segmentation suffix file name
-myelin_suffix = Path('_seg-myelin.png')         # myelin segmentation suffix file name
+axonmyelin_suffix = Path('_seg-axonmyelin-manual_pred.png') # axon + myelin segmentation suffix file name
+axon_suffix = Path('_seg-axon-manual_pred.png')             # axon segmentation suffix file name
+myelin_suffix = Path('_seg-myelin-manual_pred.png')         # myelin segmentation suffix file name
 index_suffix = Path('_index.png')         # image with the index of the axons
-axonmyelin_index_suffix = Path('_axonmyelin_index.png') # Colored axonmyelin segmentation + the index image
+axonmyelin_index_suffix = Path('_axonmyelin-manual_pred_index.png') # Colored axonmyelin segmentation + the index image
```

Comment:
```suggestion
axonmyelin_index_suffix = Path('_seg_axonmyelin-manual_pred_index.png') # Colored axonmyelin segmentation + the index image
```
For the current naming convention in this PR, we would need to add `_seg` here for consistency.
I suggest to not do the change for now, because I'll have some other news about suffixes that could change our strategy.

Reply:
Ok, on hold